### PR TITLE
Create utility for csv parsing #3

### PR DESCRIPTION
**Pull request by sahusantosh2097**

### CODE REVIEW

**Comment by sahusantosh2097** [comment]
Created the rest API for uploading the csv and parsing it and converting into java object.

File: employee-skills-service/src/main/java/org/util/employeeskillsservice/model/EmployeeInformation.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	}
+	public List<Skills> getSkills() {
+		return skills;
+	}
+	public void setSkills(List<Skills> skills) {
+		this.skills = skills;
+	}
+
+	@Override
+	public String toString() {
```
Comment (Ajinkya-Dhote) [inline employee-skills-service/src/main/java/org/util/employeeskillsservice/model/EmployeeInformation.java:108]: change this to print json object

File: employee-skills-service/src/main/java/org/util/employeeskillsservice/model/EmployeeInformation.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
+package org.util.employeeskillsservice.model;
+
+import java.util.List;
+
+import com.opencsv.bean.CsvBindByName;
+
+public class EmployeeInformation {
+    @CsvBindByName(column = "Name")
```
Comment (Ajinkya-Dhote) [inline employee-skills-service/src/main/java/org/util/employeeskillsservice/model/EmployeeInformation.java:8]: fix indentation

File: employee-skills-service/src/main/java/org/util/employeeskillsservice/model/Skills.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	}
+	public String getSkills() {
+		return skills;
+	}
+	public Skills setSkills(String skills) {
+		this.skills = skills;
+		return this;
+	}
+	@Override
+	public String toString() {
```
Comment (Ajinkya-Dhote) [inline employee-skills-service/src/main/java/org/util/employeeskillsservice/model/Skills.java:23]: change this to print json

File: employee-skills-service/src/main/java/org/util/employeeskillsservice/utility/Util.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+		questioner.add("Which of the following environments you are most comfortable using?");
+		questioner.add("Which version control tool you are aware of and can consider using?");
+		questioner.add("Which of the following frameworks, libraries, or technologies you are already proficient in?");
+		questioner.add("Which of the following languages you are already proficient in?");
+		questioner.add("List all design patterns that you understand well and can implement independently.");
+		questioner.add("Which architectural styles you understand well?");
+		questioner.add("Which DevOps & Automation tools are you familiar with and also competent with using?");
+		questioner.add("What certifications have you completed?");
+		questioner.add("What online courses have you completed?");
+        return questioner;
```
Comment (Ajinkya-Dhote) [inline employee-skills-service/src/main/java/org/util/employeeskillsservice/utility/Util.java:20]: Change logic to use column no

File: employee-skills-service/src/main/java/org/util/employeeskillsservice/service/EmployeeSkillUploaderService.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+					EmployeeInformation::getEmployeeCode, Function.identity()));
+			employessIdMapping.forEach((key, value) -> {
+				value.setSkills(employeeSkillMapping.get(key));
+			});
+            return employessIdMapping.values();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
```
Comment (Ajinkya-Dhote) [inline employee-skills-service/src/main/java/org/util/employeeskillsservice/service/EmployeeSkillUploaderService.java:72]: simplify this logic

File: employee-skills-service/src/main/java/org/util/employeeskillsservice/model/Skills.java [hunk 6]
```diff
@@ -0,0 +1,26 @@
+package org.util.employeeskillsservice.model;
+
+public class Skills {
+
+	private long employeeId;
+	private String skills;
```
Comment (Ajinkya-Dhote) [inline employee-skills-service/src/main/java/org/util/employeeskillsservice/model/Skills.java:6]: add a filed called tag

File: employee-skills-service/src/main/java/org/util/employeeskillsservice/utility/Util.java (hunk 4 above)
Comment (sahusantosh2097) [reply employee-skills-service/src/main/java/org/util/employeeskillsservice/utility/Util.java:20]: Kept the column name

File: employee-skills-service/src/main/java/org/util/employeeskillsservice/model/Skills.java (hunk 6 above)
Comment (sahusantosh2097) [reply employee-skills-service/src/main/java/org/util/employeeskillsservice/model/Skills.java:6]: Yes done.

**Pull request closed without merging**